cli/azd/pkg/azdo/pipeline.go

```diff
@@ -154,6 +154,12 @@ func getDefinitionVariables(
 		"AZURE_SUBSCRIPTION_ID":    createBuildDefinitionVariable(credentials.SubscriptionId, false, false),
 	}
 
+	if provisioningProvider.Provider == provisioning.Bicep {
+		if v, has := env.LookupEnv(environment.ResourceGroupEnvVarName); has {
```

Comment:
Nit: Rename to something more discoverable.
```suggestion
		if resourceGroupName, has := env.LookupEnv(environment.ResourceGroupEnvVarName); has {
```

Reply:
Went with `rgName` which is descriptive enough and keeps me from running afoul of the line length limit.